lib/matplotlib/testing/nose/plugins/conversion_cache.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from pprint import pprint
+from nose.plugins import Plugin
```

Comment:
We don't depend on nose anymore. I am not sure how you convert this to a pytest compatible plugin.